src/test/java/org/kairosdb/datastore/h2/H2DatastoreTest.java

```diff
@@ -127,6 +127,8 @@ public void test_queryDatabase_noMetric() throws DatastoreException
 	public void test_serviceKeyStore_singleService()
 			throws DatastoreException
 	{
+		// NOTE (wittekm, July 2020) - This test appears to be flaky; we should dependency-inject a Time Provider.
+		// I will do this ASAP in a separate diff
```

Comment:
If it's not too much work, then please go ahead.

However, if it proves difficult, it may not be worthwhile. Personally, I would like to remove all the non-Cassandra datastore support from this fork to minimize the maintenance of this package and let us focus on what it does best.

_Background:_ In days before Docker, having an embedded data store as an option was essential to allow for functional testing of code. It was difficult and cumbersome to require developers setup and run Cassandra instances in order to execute tests. However, with Docker (frankly, with Fabric8's fantastic Maven plugin for Docker) it's now super easy to provide repeatable supporting infrastructure for functional tests. This actually improves test accuracy since it means more tests execute against the actual backend used in production. I do not see any reason to run H2 ever as a backend for KDB in production and since it's no longer needed for development/testing, why continue to maintain the complexity? (FYI @BrandonArp)